### ISSUE TITLE
Broken no compatible source was found video window on Firefox 79

### ISSUE THREAD

**Issue by mrpau-richard** [learningequality/kolibri#7411]
### Observed behavior
Broken no compatible source was found video window on Firefox 79. 

This is the comparison between Firefox and Chrome with the same video content. 
![gif-temp](https://user-images.githubusercontent.com/8663934/88920364-4c4d7880-d29f-11ea-90a0-1c944c20ffc6.gif)

### Expected behavior

…

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce
* Assuming that  Kolibri 0.14.0-beta14 is installed and running at Ubuntu 16.04 LTS
* Do not install the [restricted-extras](https://help.ubuntu.com/community/RestrictedFormats) package
* Log in as super admin user and import video content
* Using Firefox 79 browser navigate at the video content.
* Observe the video window layout. 

### Context

Tell us about your environment, including:
 * Kolibri version: 0.14.0-beta14
 * Operating system: Ubuntu 16.04 LTS
 * Browser: Firefox 79

**Comment by radinamatic** [comment]
@mrpau-richard Did you install the [`restricted-extras`](https://help.ubuntu.com/community/RestrictedFormats) package on that Ubuntu VM?

**Comment by mrpau-richard** [comment]
I did not install the [restricted-extras](https://help.ubuntu.com/community/RestrictedFormats) when I'm having this issue @radinamatic. The issue I'm pointing is the video window is broken when that package is not installed. 

The video will played properly after I installed the package.
![Screen Shot 2020-07-31 at 3 54 30 PM](https://user-images.githubusercontent.com/8663934/89013254-27601080-d346-11ea-9a3d-fc0c09dd91b6.png)
